lib/alchemy/admin/locale.rb

```diff
@@ -46,15 +46,16 @@ def available_locale
       def locale_from_user
         return if !current_alchemy_user
         if user_has_preferred_language?
-          current_alchemy_user.language
+          current_alchemy_user.language.to_sym
```

Comment:
But `set_translation` [calls `available_locale`](https://github.com/IngoAlbers/alchemy_cms/blob/141b6edc9f75e0cc6572aaa67e485e49442cc754/lib/alchemy/admin/locale.rb#L20) which [calls `to_sym` on the `locale_from_user`](https://github.com/IngoAlbers/alchemy_cms/blob/141b6edc9f75e0cc6572aaa67e485e49442cc754/lib/alchemy/admin/locale.rb#L42).

I just want to avoid double `to_sym` calls ;)

Does it break for you, if you don't add `to_sym` to `locale_from_user`?